packages/11ty/_layouts/table-of-contents.11ty.js

```diff
@@ -36,6 +37,8 @@ module.exports = class TableOfContents {
       : ''
     const containerClass = presentation === 'grid' ? 'is-fullhd' : ''
 
+    const biblio = page_bibliography || config.params.pageBibliography
+
```

Comment:
I feel like mixing liquid syntax and JS string interpolation is a little funky, I think this might read better if all the conditional logic is in JS:

```suggestion
    const bibliography = page_bibliography || config.params.pageBibliography
      ? '{% bibliography pageReferences %}'
      : ''

```